javascript/index.js

```diff
@@ -1,48 +1,61 @@
-var typed = new Typed('#element', {
-  strings: ['Projects', 'Community', 'Innovation', 'Contribution'],
+var typed = new Typed("#element", {
+  strings: ["Projects", "Community", "Innovation", "Contribution"],
   typeSpeed: 50,
 });
 
-let contact = JSON.parse(localStorage.getItem('contact')) || {
-  name : '',
-  email: '',
-  message: ''
-};
-
-document.getElementById('name').value = contact.name;
-document.getElementById('email').value = contact.email;
-document.getElementById('message').value = contact.message;
-
-function handleFormSubmit(event) {
-  
-  event.preventDefault(); // prevents reloading of website.
-  
-  contact.name = document.getElementById('name').value;
-  contact.email = document.getElementById('email').value;
-  contact.message = document.getElementById('message').value;
-
-  localStorage.setItem('contact', JSON.stringify(contact));
-
-}
-
-const formElement = document.getElementById('messageForm');
-
-formElement.addEventListener('submit', handleFormSubmit);
-
-const elements = document.getElementsByClassName('blur');
+const elements = document.getElementsByClassName("blur");
 
 function toggle(x) {
   if (x) {
     for (let i = 0; i < elements.length; i++) {
-      elements[i].classList.add('blurred');
+      elements[i].classList.add("blurred");
     }
-    document.getElementById('popup').classList.add('active');  
-    document.getElementById('contact').scrollIntoView();  
-  }
-  else {
+    document.getElementById("popup").classList.add("active");
+    document.getElementById("contact").scrollIntoView();
+  } else {
     for (let i = 0; i < elements.length; i++) {
-      elements[i].classList.remove('blurred');
+      elements[i].classList.remove("blurred");
     }
-    document.getElementById('popup').classList.remove('active');
+    document.getElementById("popup").classList.remove("active");
   }
-}
\ No newline at end of file
+}
+
+var form = document.getElementById("messageForm");
+
+async function handleSubmit(event) {
+  event.preventDefault();
+  const status = document.getElementById("form-status");
+  var data = new FormData(event.target);
+  fetch(event.target.action, {
+    method: form.method,
+    body: data,
+    headers: {
+      Accept: "application/json",
+    },
+  })
+    .then((response) => {
+      if (response.ok) {
+        status.innerHTML = "Thanks for your submission!";
+        toggle("body");
+        form.reset();
+      } else {
+        response.json().then((data) => {
+          if (Object.hasOwn(data, "errors")) {
+            status.innerHTML = data["errors"]
+              .map((error) => error["message"])
+              .join(", ");
+            toggle("body");
+          } else {
+            status.innerHTML =
+              "Oops! Too many requests. Please try again later.";
+            toggle("body");
+          }
+        });
+      }
+    })
+    .catch((error) => {
+      status.innerHTML = "Oops! There was a problem submitting your form";
```

Comment:
How about suggesting the user to use other platforms to contact us.
- linkedIn page 
- GitHub Discussion 

Your suggestions on  - A `mailto:` button  on unsuccessful form submission

Reply:
Yes, we can use linkedIn page or github discussion in the pop up that occurs when form is submitted.
Now, mailto button feature will be pretty good, so that user can interact with us. 

You can merge this branch, therefore I will continue the updates or suggest me what should I do..